catalogmanagementv1/catalog_management_v1_integration_test.go

```diff
@@ -3346,6 +3501,22 @@ var _ = Describe(`CatalogManagementV1 Integration Tests`, func() {
 		})
 	})
 
+	// Ounset pc managed
```

Comment:
This is a typo here, right?
```suggestion
	// Unset pc managed
```